debian package build depends on non-existant LICENSE-2.0.txt

Copying WALinuxAgent.egg-info to /tmp/buildd/walinuxagent-1.3/debian/tmp/usr/lib/python2.7/dist-packages/WALinuxAgent-2.1.6.1.egg-info
    running install_scripts
       dh_install
       dh_installdocs
    cp: cannot stat 'LICENSE-2.0.txt': No such file or directory
    dh_installdocs: cp -a LICENSE-2.0.txt debian/walinuxagent/usr/share/doc/walinuxagent returned exit code 1
    make: *** [binary] Error 2
    dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
    E: Failed autobuilding of package
    I: unmounting dev/pts filesystem
    I: unmounting run/shm filesystem
    I: unmounting proc filesystem
    I: cleaning the build env
    I: removing directory /var/cache/pbuilder/build//16069 and its subdirectories

--- a/debian/docs
+++ b/debian/docs
@@ -1,3 +1,3 @@
 NOTICE
-LICENSE-2.0.txt
+LICENSE.txt
 README.md

